lib/ext/heroku/command/addons.rb

```diff
@@ -174,7 +174,14 @@ def create
 
       action("Creating #{addon['name'].downcase}") {}
       action("Adding #{addon['name'].downcase} to #{app}") {}
-      action("Setting #{addon['config_vars'].join(', ')} and restarting #{app}") do
+
+      if addon.has_key? 'config_vars' && addon['config_vars']
```

Comment:
How about

```ruby
if addon.fetch('config_vars', false)
  ...
```

It succeeds if the `config_vars` key is there and has a value, fails if the key or its value are missing